### DISCUSSION TITLE
请问如何解决[Errno 99] Cannot assign requested address

### DISCUSSION THREAD

**Discussion by TweakyLemming66** [Lost-MSth/Arcaea-server#26] [Q&A]
如题，于Ubuntu服务器上运行时出现此报错并退出，萌新求解决方法

![A0EAC460-0E7D-4A12-BA33-545E6F6EC80D](https://user-images.githubusercontent.com/85770086/125301204-51df8980-e35d-11eb-8e0c-c6426ea2e197.png)

提前感谢

**Comment by Lost-MSth** [comment]
看起来像是端口冲突，换个端口试试

**Reply by TweakyLemming66** [reply]
额能顺便问一下Charles怎么用吗，没接触过这个软件

**Reply by Lost-MSth** [reply]
最好的方法是问百度()

毕竟我也不会用Linux下的Charles

**Reply by TweakyLemming66** [reply]
> 最好的方法是问百度()
> 
> 毕竟我也不会用Linux下的Charles

好吧（
